tck/faces-tck/pom.xml

```diff
@@ -123,6 +123,10 @@
                         </goals>
                         <configuration>
                             <executable>docker/build_oldtck.sh</executable>
+                            <arguments>
+                                <argument>${maven.multiModuleProjectDirectory}</argument> 
```

Comment:
Note that this property is set just in the mvn shell script and it is discouraged to use it - it may be removed in future versions.

Reply:
True, maybe handle this in a new PR?